core/domain/feedback_services_test.py

```diff
@@ -357,6 +367,83 @@ def test_get_total_open_thread_for_single_exploration(self):
         self.assertEqual(thread.num_open_threads, 1)
         self.assertEqual(thread.num_total_threads, 1)
 
+    def test_get_next_page_of_all_feedback_messages(self):
+        self.save_new_default_exploration(self.EXP_ID_1, self.EXP_ID_2)
+        suggestion_services.create_suggestion(
+            feconf.SUGGESTION_TYPE_EDIT_STATE_CONTENT,
+            feconf.ENTITY_TYPE_EXPLORATION,
+            self.EXP_ID_1,
+            1,
+            self.user_id,
+            {
+                'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+                'state_name': 'state',
+                'property_name': exp_domain.STATE_PROPERTY_CONTENT,
+                'new_value': 'new content'
+            },
+            'some text')
+        thread_id = feedback_services.get_threads(
+            feconf.ENTITY_TYPE_EXPLORATION, self.EXP_ID_1
+        )[0].id
+        feedback_services.create_message(
+            thread_id, self.user_id, None, None, 'some text')
+        feedback_services.create_message(
+            thread_id, self.user_id, None, None, 'Another text')
+        messages_on_page = feedback_services.get_messages(thread_id)
+        dictionary_list_message_on_page_1 = []
+        dictionary_list_message_on_page_2 = []
```

Comment:
These names make me think one list has the messages from page 1 and the other list has messages from page 2, but I think you actually mean that one has lists from `get_next_page_of_all_feedback_messages` and the other has the list from `get_messages`

Reply:
Will change the names to something suitable.